reconstruction/recoiltof/pom.xml

```
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <project xmlns="http://maven.apache.org/POM/4.0.0"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.jlab.clas12.detector</groupId>
+  <artifactId>clas12detector-recoiltof</artifactId>
+  <version>13.4.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.jlab.clas12</groupId>
+    <artifactId>reconstruction</artifactId>
+    <version>13.4.0-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>clas-detector</artifactId>
+      <version>13.4.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>clas-io</artifactId>
+      <version>13.4.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>clas-geometry</artifactId>
+      <version>13.4.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>clas-reco</artifactId>
+      <version>13.4.0-SNAPSHOT</version>
+    </dependency>
+    
+  </dependencies>
```

From the [CI dependency analysis errors](https://github.com/JeffersonLab/coatjava/actions/runs/18953615623/job/54125074713?pr=921#step:7:1699), you just need to do what it complains about: remove the unused dependencies.

```suggestion
  <dependencies>

    <dependency>
      <groupId>org.jlab.clas</groupId>
      <artifactId>clas-io</artifactId>
      <version>13.4.0-SNAPSHOT</version>
    </dependency>
    <dependency>
      <groupId>org.jlab.clas</groupId>
      <artifactId>clas-reco</artifactId>
      <version>13.4.0-SNAPSHOT</version>
    </dependency>
    
  </dependencies>
```